src/containers/account/AccountContainer.js

```diff
@@ -94,30 +113,30 @@ const AccountContainer = () => {
                 onClick={() => clipboardWriteText(AuthUtils.getAuthToken())} />
           </ActionsGrid>
         </StackGrid>
-        {
-          getAtlasCredentialsRS === RequestStates.SUCCESS && (
-            <>
-              <StackGrid gap={4}>
-                <Typography component="h2" variant="body2">ATLAS USERNAME</Typography>
-                <ActionsGrid fit>
-                  {PasswordInput}
-                  <CopyButton
-                      aria-label="copy atlas username"
-                      onClick={() => clipboardWriteText(atlasCredentials.get('username'))} />
-                </ActionsGrid>
-              </StackGrid>
-              <StackGrid gap={4}>
-                <Typography component="h2" variant="body2">ATLAS CREDENTIAL</Typography>
-                <ActionsGrid fit>
-                  {PasswordInput}
-                  <CopyButton
-                      aria-label="copy atlas credential"
-                      onClick={() => clipboardWriteText(atlasCredentials.get('credential'))} />
-                </ActionsGrid>
-              </StackGrid>
-            </>
-          )
-        }
+        <StackGrid gap={4}>
+          <Typography component="h2" variant="body2">ATLAS USERNAME</Typography>
+          <ActionsGrid fit>
+            {PasswordInput}
+            <CopyButton
+                aria-label="copy atlas username"
+                isDisabled={atlasCredentialsPending}
+                onClick={() => clipboardWriteText(atlasCredentials.get('username'))} />
+          </ActionsGrid>
+        </StackGrid>
+        <StackGrid gap={4}>
+          <Typography component="h2" variant="body2">ATLAS CREDENTIAL</Typography>
+          <ActionsGrid fit>
+            {PasswordInput}
+            <CopyButton
+                aria-label="copy atlas credential"
+                isDisabled={atlasCredentialsPending}
+                onClick={() => clipboardWriteText(atlasCredentials.get('credential'))} />
+            <RefreshButton
+                aria-label="regenerate atlas credential"
+                isDisabled={atlasCredentialsPending}
+                onClick={regenerateAtlasCredential} />
```

Comment:
I think you should use the `isLoading` prop to show a spinner when `regenerateAtlasCredentialsRS` is pending.